src/app.py

```diff
@@ -506,6 +506,20 @@ def _validate_token_if_auth_header_exists(request):
         if isinstance(user_info, Response):
             unauthorized_error(user_info.get_data().decode())
 
+
+# Use the Flask request.args MultiDict to see if 'reindex' is a URL parameter passed in with the
+# request and if it indicates reindexing should be supressed. Default to reindexing in all other cases.
+def _suppress_reindex() -> bool:
+    if 'reindex' not in request.args:
+        return False
+    reindex_str = request.args.get('reindex').lower()
+    if reindex_str == 'false':
+        return True
+    elif reindex_str == 'true':
+        return False
+    raise Exception(f"The value of the 'reindex' parameter must be True or False."
```

Comment:
Can we also explicitly specify that the value is case-insensitive in the log message? Saying `must be True or False` is only one case.